src/client/containers/ProfilePage/AdminProfile/AdminProfile.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState, useRef } from 'react';
+
+import admin from '../../../assets/images/admin.png';
+import { Layout } from '../../../components';
+import { AppHeader } from '../../../components/Appheader/AppHeader.component';
+import { Avatar } from '../../../components/Avatar/Avatar';
+import Footer from '../../../components/footer/footer';
+import Header from '../../../components/Heading/Heading';
+import Input from '../../../components/Input/Input';
+import { useAuthenticatedFetch } from '../../../hooks/useAuthenticatedFetch';
+
+export const AdminProfilePage = () => {
+  const [adminName, setAdminName] = useState(null);
+  const [adminRole, setAdminRole] = useState(null);
+  const [editableName, setEditableName] = useState(adminName);
+  const [editableRole, setEditableRole] = useState(adminRole);
```

Comment:
There is no need to have all four of these, you only need two.
Also, prefer naming like `name` to `editableName`.